src/manage/permissions.md

```diff
@@ -1,49 +1,89 @@
 :::
 
-## Workspace member roles
-**Members** of a workspace have access to all projects in a workspace. The level of access to the project is defined by their member roles:
-- **Reader**: users with **read** permission are able to see the list of projects, projects data and [history](./project-details.md)
-- **Writer**: in addition to the **Read** access, users can also do changes in the projects: [adding/removing/updating features](../field/mobile-features/), change [layer settings and project properties](../gis/features/). Users who are expected to actively contribute to the project need to have the **write** permission. 
-- **Admin**: in addition to the **Write** access, users with this level of access can [delete the project](./project-advanced/#delete-a-project) or [transfer](./project-advanced/#transfer-a-project) it to another workspace. Admins can also create new projects and manage workspace members.
-- **Owner**: in addition to the **Admin** role, owners have access to [invoicing and subscription](./subscriptions/) settings.
+## Workspace member roles and project permissions
+Member roles and project permissions present similar options in terms of what a user can or cannot do. The difference is whether these options are defined on a workspace or on a project level. Compare the overview of [workspace member roles](#workspace-member-roles-overview) and of [project permissions](#project-permissions-overview) for more details.
 
-| | Reader  | Writer | Admin  | Owner  |
+- **Reader**: users with **Read** permission are able to see projects, projects data and [history](../project-details)
+- **Editor** (available on <MainPlatformNameLink /> Cloud and <EnterprisePlatformNameLink />): in addition to the **Read** access, users can [add/remove/update features](../field/mobile-features/). They *cannot* make changes in the project properties, add/remove fields in GeoPackage layers or remove GeoPackage files from the project. Editors can:
+	- *add* files such as photos, shapefiles, GeoPackage files - any file except for `*.qgs`, `*.qgz`, `mergin-config.json`
+	- *edit* files except for `*.qgs`, `*.qgz`, `mergin-config.json` and non-diff based `*.gpkg` changes
+	- *remove* files except for `*.qgs`, `*.qgz`, `mergin-config.json` and `*.gpkg`
+- **Writer**: in addition to the **Editor** access, users can also change [layer settings and project properties](../gis/features/).
+- **Owner**: in addition to the **Write** access, users with this level of access can [delete the project](./project-advanced/#delete-a-project) or [transfer](./project-advanced/#transfer-a-project) it to another workspace. Owners can also create new projects and manage workspace members. Owners have also access to [invoicing and subscription](./subscriptions/) settings.
+- **Admin**: This role is only available for workspace members. It is similar to **Owner**, but without access to invoicing and subscriptions.
```

Comment:
Can we first mention `Admin` and only after that `Owner`? It seems like a natural flow, growing from the _lowest_ to the _highest_